crate/operator/create.py

```diff
@@ -611,7 +612,7 @@ def get_statefulset_init_containers(crate_image: str) -> List[V1Container]:
             image="busybox:1.35.0",
             image_pull_policy="IfNotPresent",
             name="init-sysctl",
-            security_context=V1SecurityContext(privileged=True),
+            security_context=V1SecurityContext(capabilities=V1Capabilities(add=["SYS_CHROOT"])),
```

Comment:
```suggestion
            security_context=V1SecurityContext(capabilities=V1Capabilities(add=["SYS_ADMIN"])),
```

While I think it is an improvement, I think this is not the place where we need `SYS_CHROOT`

Reply:
Thank you, is there any particular operation that comes to mind for which we need `SYS_ADMIN` ?
Do you think the CI tests in this repo are comprehensive enough or you think a change here would need to be tested in some other way?

Reply:
Discussed with @WalBeh , this container should be fine with `SYS_ADMIN` as it runs `sysctl`, it is the CrateDB container that needs `SYS_CHROOT`